docs/wiki/research/scaling/core-changes.md

```diff
@@ -0,0 +1,47 @@
+# Changes to Ethereum core to accomplish scalability
+
+## Introduction
+
+L2 Rollups need to enable the permissionless reconstruction of the L2 state. To achieve this, they must ensure Data Availability, which refers to the global ordering of inputs for transactions that have altered the state.
+Until EIP-4844 goes live on the Mainnet (expected in March 2024), Rollups have been storing Data Availability within a section of the transaction known as "calldata". Currently, this "calldata" represents the primary bottleneck for scaling Ethereum, as storing data in calldata is expensive.
```

Comment:
to have this future proof, you can just phrase it as 'before 4844 was live on maiinnet, rollups have been storing data only via calldata..' 

there is an interesting resource you could add, it's now outdated but there might be other similar dashboard. It's measuring how much L2 data is stored in mainnet blockchain https://dune.com/barnabe/Rollup-statistics